kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerExecutor.java

```diff
@@ -89,6 +92,8 @@ public void stopKieServer() {
             throw new RuntimeException("Kie execution server is already stopped!");
         }
         kieServer.destroy();
+        // The KieServices instance that was seen by the kieserver, will never be seen again at this point
+        ((KieServicesImpl) KieServices.Factory.get()).nullAllContainerIds();
```

Comment:
@mswiderski @sutaakar performing the fix on the test side as discussed, as from a classloading point of view the KieServices instance seen by the kie-server will be gone and should not be seen by a new kie-server started (created). May I have your confirmation this is indeed the intended behavior of the test, please?